### ISSUE TITLE
Unable to install new version into IIS

### ISSUE THREAD

**Issue by Omzig** [ironmansoftware/powershell-universal#1823]
### Steps to Reproduce

The more i test this, the more i think it is a bug.
https://forums.ironmansoftware.com/t/nested-iis-opens-with-a-blank-page/8283

### Expected behavior

```console
a working app
```

### Actual behavior

```console
Request URL: https://dev.psu.domain.com/psuSandbox/api/v1/accessible
Request Method: GET
Status Code: 401 Unauthorized

Request URL: https://dev.psu.domain.com/psuSandbox/api/v1/gitstatus/edit
Request Method: GET
Status Code: 401 Unauthorized
```

### Environment data

PSU: 3.6.0+

### Visuals

![image](https://user-images.githubusercontent.com/1987296/207636267-fe6e1446-399b-474d-9688-9254c60729ae.png)

# Failed Workarounds
- Copy the contents of the /Universal app from the MIS into the IIS folder and start the app pool. v3.6.1
- down grade to 3.6.0 (3680976368)
# Successful Workarounds
- Downgrade to 3.5.5
- Downgrade to 3.5.6

**Comment by adamdriscoll** [comment]
This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:

https://forums.ironmansoftware.com/t/fatal-nested-iis-opens-with-a-blank-page/8283/1

**Comment by Omzig** [comment]
> This issue has been mentioned on **Ironman Software Forums**. There might be relevant details there:
> 
> https://forums.ironmansoftware.com/t/fatal-nested-iis-opens-with-a-blank-page/8283/1

that is me, and i put the link at the top of this issue, it started as a question, then turned into a bug...

**Comment by adamdriscoll** [comment]
@Omzig - Can you let me know a little more about this configuration? 

I just tried this myself and can load PSU in IIS with Windows auth enabled as well as forms auth in 3.6.x. 

I'm also curious if there is a browser error you could share? If you press F12 in the browser it should give a hint as to why the page is white. 

Can you also try Ctrl+F5 and\or clearing the application data in the dev tools? 

<img width="274" alt="image" src="https://user-images.githubusercontent.com/1256531/207640216-92900f85-2764-4910-9198-e54e54cd2fa0.png">

**Comment by adamdriscoll** [comment]
Never mind. I think I've reproduced this.

**Comment by adamdriscoll** [comment]
@Omzig - This has been resolved and is in nightly build 3697129904. Could you please verify in your environment as well? https://ironmansoftware.com/release/powershell-universal-nightly

**Comment by Omzig** [comment]
> @Omzig - This has been resolved and is in nightly build 3697129904. Could you please verify in your environment as well? https://ironmansoftware.com/release/powershell-universal-nightly

Success
![image](https://user-images.githubusercontent.com/1987296/207730927-5d1c54c3-c4c4-427f-b7dd-007c3128c056.png)

**Comment by adamdriscoll** [comment]
Fantastic. Thanks for trying that. We'll get a release out.
